File: rock-paper-scissors/src/main/java/cocharge/task/rockpaperscissors/RockPaperScissorsApplication.java
package cocharge.task.rockpaperscissors;

import cocharge.task.rockpaperscissors.Engine.Flow;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class RockPaperScissorsApplication {

    public static void main(String[] args) throws InterruptedException {
        SpringApplication.run(RockPaperScissorsApplication.class, args);
        Flow gameFlow = new Flow();
        gameFlow.startGame();
    }

}

File: rock-paper-scissors/src/main/java/cocharge/task/rockpaperscissors/RockPaperScissorsApplication.kt
package cocharge.task.rockpaperscissors

import cocharge.task.rockpaperscissors.Engine.Flow
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


class RockPaperScissorsApplication

fun main(args: Array<String>) {
        runApplication<RockPaperScissorsApplication>(*args)
val gameFlow = Flow()
    gameFlow.startGame()
}